.yarnrc.yml

```diff
@@ -3,5 +3,9 @@ nodeLinker: node-modules
 plugins:
   - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
     spec: "@yarnpkg/plugin-workspace-tools"
+  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
+    spec: "@yarnpkg/plugin-interactive-tools"
```

Comment:
FYI: I have added this in order to be able to run [`yarn upgrade-interactive`](https://classic.yarnpkg.com/lang/en/docs/cli/upgrade-interactive/)